testsuite/openshift/client.py

```diff
@@ -73,7 +73,7 @@ def project(self):
     def connected(self):
         """Returns True, if user is logged in and the project exists"""
         try:
-            self.do_action("status")
+            self.do_action("get", "ns", self._project)
```

Comment:
kubectl/oc has command for checking privileges. https://kubernetes.io/docs/reference/kubectl/generated/kubectl_auth/kubectl_auth_can-i/
Not sure which is the best but some candidate examples from the docs are:
`auth can-i '*' '*' `
`auth can-i create pods --all-namespaces`

Reply:
I think that would be similar hacky solution as I used, point of the function to determine that you are connected and the namespace exists and both of these solution achieve by "accident", their purpose is to get a different information